### ISSUE TITLE
Typo in README

### ISSUE THREAD

**Issue by maxrobot** [pandoc/pandoc-action-example#7]
Chaps, 

In the readme for the long example,

```
name: Long Usage

on: push

jobs:
  convert_via_pandoc:
    runs-on: ubuntu-18.04
    steps:
      - run: echo "foo" > input.txt  # create an example file
      - uses: docker://pandoc/core:2.9
        with:
          args: |>  # allows you to break string into multiple lines
            --standalone
            --output=index.html
            input.txt
```

I believe `args: |>` should be `args: >-`.

**Comment by mcaci** [comment]
Indeed I was following the guide while trying the long usage example in my own work and I discovered that the `args: |>` is not correct and that this `args: >-` was the syntax to use. Even if it's not a great contribution I opened the aforementioned PR to correct this. Hope this helps even though it's a small change.

**Comment by alerque** [comment]
This has been corrected in the readme thanks to @mcaci in #10. The live example was correct, just the readme had bogus syntax.
